src/elements/content-sidebar/versions/__tests__/VersionsSidebarContainer.test.js

```diff
@@ -69,7 +69,7 @@ describe('elements/content-sidebar/versions/VersionsSidebarContainer', () => {
         });
     });
 
-    describe('handleActionDelete', () => {
+    describe(' Delete', () => {
```

Comment:
Is this change needed? If not, can we revert it?

Reply:
👍  That was an accidental change I think. I have reverted it